src/repo/zcl_abapgit_repo_srv.clas.abap

```diff
@@ -15,7 +15,6 @@ CLASS zcl_abapgit_repo_srv DEFINITION
       IMPORTING
         ii_srv TYPE REF TO zif_abapgit_repo_srv.
 
-  PROTECTED SECTION.
```

Comment:
SE80 will add this back in again

Reply:
one of my exits accidentally removed empty sections. i put them back